cloudbuild-test.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+  - name: 'gcr.io/cloud-builders/gcloud'
+    script: |
+      #!/usr/bin/env bash
+      set -e
+      REPO_TOKEN="$(gcloud auth print-access-token)"
+      cat > ~/.npmrc <<EOL
+      registry=https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:always-auth=true
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:_authToken=${REPO_TOKEN}
+      EOL
```

Comment:
Similarly with this, we should really be reading from the same npmrc file for both testing and publishing

Reply:
Each build have its own environment. token will be generated separately for each build. I am not sure what do you mean here by having same .npmrc file

Reply:
If there are different files, then they're likely to diverge. Which means we would be a different setup for testing and releasing - which makes it more likely we're going to publish the wrong thing at some point

Reply:
We can leave it for now. Also these are slightly different files. for publish we need NPM token for testing we do not. We can create a single script for creating both in future may be.